dart/packages/fury-core/lib/src/annotation/fury_key.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:meta/meta_meta.dart';
+
+@Target({TargetKind.field})
+class FuryKey {
```

Comment:
Could we add docs/comments for this class? Without commens, it's not easy to understand what is it used for